src/index.ts

```diff
@@ -112,17 +124,17 @@ const showPrivacyManager = () => {
 export const cmp: CMP = (() => {
 	if (isServerSide) {
 		return {
-			init: () => void 0,
-			showPrivacyManager: () => void 0,
-			willShowPrivacyMessage: () => new Promise(() => false),
-			willShowPrivacyMessageSync: () => false,
-			hasInitialised: () => true,
-			version: __PACKAGE_VERSION__,
-
-			// special helper methods for disabling CMP
-			__isDisabled: isDisabled,
-			__enable: enable,
-			__disable: disable,
+			__disable: serverSideWarn,
+			__enable: serverSideWarnAndReturn(false),
+			__isDisabled: serverSideWarnAndReturn(false),
+			hasInitialised: serverSideWarnAndReturn(false),
+			init: serverSideWarn,
+			showPrivacyManager: serverSideWarn,
+			version: 'n/a',
+			willShowPrivacyMessage: serverSideWarnAndReturn(
+				Promise.resolve(false),
+			),
+			willShowPrivacyMessageSync: serverSideWarnAndReturn(false),
 		} as CMP;
```

Comment:
You should be able to remove the assertion!
```suggestion
		};
```